src/Microsoft.Health.Dicom.SqlServer/Features/Schema/Sql/Sprocs/GetTotalAndSumContentLengthIndexedAsyncV57.sql

```diff
@@ -0,0 +1,19 @@
+/***************************************************************************************/
+-- STORED PROCEDURE
+--     GetTotalAndSumContentLengthIndexedAsyncV57
+--
+-- FIRST SCHEMA VERSION
+--     57
+--
+-- DESCRIPTION
+--     Retrieves total sum of content length across all FileProperty rows
+--
+/***************************************************************************************/
+CREATE OR ALTER PROCEDURE dbo.GetTotalAndSumContentLengthIndexedAsyncV57
```

Comment:
nit: you can create with GetTotalAndSumContentLengthIndexedAsync without the version since this is the first one